src/widgets/JSONForm/JSONForm.js

```diff
@@ -147,6 +174,8 @@ export const JSONFormComponent = React.forwardRef(
   ) => {
     if (!surveySchema) return null;
 
+    const [hasSchemaError, setSchemaError] = useState(false);
+
     const { uiSchema, jsonSchema } = surveySchema;
     const validator = useMemo(() => ajv.compile(jsonSchema), [jsonSchema]);
```

Comment:
I've also found that this line can throw an error if the schema is super invalid (say if "required" is not defined as an array)

E.g.
`{"type": "object", "title": "ADR", "required": "not-exist"}`

Reply:
Ohhh nice, thanks!

0a4e851

Also thanks for the grammer lessing, I appreciate !